src/main/resources/stock_trades_schema.avro

```diff
@@ -0,0 +1,64 @@
+{
+  "namespace": "ksql",
+  "name": "StockTrade",
+  "doc": "Defines a hypothetical stock trade using some known simulated stock symbols.",
```

Comment:
"known" and "simulated": seem contradictory?  For consideration:

```
Defines a hypothetical stock trade using fake stock symbols.
```

Reply:
"known simulated stock symbols" meaning they are known as simulated in the industry.

Reply:
Can we call them "known test stock symbols" ... reference: https://iextrading.com/trading/alerts/2017/027/